quic/s2n-quic-tls/src/server.rs

```diff
@@ -103,6 +104,50 @@ impl Builder {
         Ok(self)
     }
 
+    pub fn with_trust_client_certificate_signed_by<C: IntoCertificate>(
```

Comment:
Should we just call this `with_trusted_certificate`? I think since it's just adding it to the trust store it doesn't have to necessarily relate to the client auth.

Reply:
I am fine with renaming it that. I guess I wasn't initially thinking the server would have a reason to consult its trust store for anything other than client auth - but you are right, it is just adding that cert to the trust store. I"ll update in a forthcoming revision.